docs/en/modules/net.md

```diff
@@ -282,3 +282,100 @@ Sets the IP of the DNS server used to resolve hostnames. Default: resolver1.open
 #### See also
 [`net.dns:getdnsserver()`](#netdnsgetdnsserver)
 
+# net.cert Module
+
+This controls certificate verification when SSL is in use. 
+
+## net.cert.verify()
+
+Controls the vertificate verification process when the Nodemcu makes a secure connection.
+
+#### Syntax
+`net.cert.verify(enable)`
+
+`net.cert.verify(pemdata)`
```

Comment:
I did the change that you suggested and don't flash the cert if it was already there. It changes the time spent in the call from 80ms down to 3ms! There isn't a way to see the cert that is stored --partly because it is just a blob, and partly because I dodn't want people to inspect the client private key (which will use all the same mechanisms).